en/05_Contributing/index.md

```diff
@@ -13,4 +13,4 @@ icon: heart
 
 For more detailed guidance, read one of the following pages:
 
-[CHILDREN]
+[CHILDREN includeFolders]
```

Comment:
This should make it so the "Coding Conventions" folder is displayed as a child card, using the index.md file to fill it in.